operator/cmd/mesh/operator-remove.go

```diff
@@ -75,17 +82,32 @@ func operatorRemove(args *rootArgs, orArgs *operatorRemoveArgs, l clog.Logger) {
 	}
 
 	l.LogAndPrintf("Removing Istio operator...")
-	reconciler, err := helmreconciler.NewHelmReconciler(client, restConfig, nil, &helmreconciler.Options{DryRun: args.dryRun, Log: l})
+	var iop *iopv1alpha1.IstioOperator
+	if orArgs.revision != "" {
+		emptyiops := &v1alpha1.IstioOperatorSpec{Profile: "empty", Revision: orArgs.revision}
```

Comment:
comment would be helpful here, otherwise hard to see why IOP is needed e.g. 
// Create an empty IOP for the purpose of populating revision. Apply code requires a non-nil IOP.

Reply:
done